eth/api_debug.go

```diff
@@ -432,3 +434,41 @@ func (api *DebugAPI) SetTrieFlushInterval(interval string) error {
 func (api *DebugAPI) GetTrieFlushInterval() string {
 	return api.eth.blockchain.GetTrieFlushInterval().String()
 }
+
+type ConversionStatusResult struct {
+	Started bool `json:"started"`
+	Ended   bool `json:"ended"`
+}
+
+func (api *DebugAPI) ConversionStatus(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) (*ConversionStatusResult, error) {
+	block, err := api.eth.APIBackend.BlockByNumberOrHash(ctx, blockNrOrHash)
+	if err != nil {
+		return nil, err
+	}
+	data, err := rawdb.ReadVerkleTransitionState(api.eth.ChainDb(), block.Root())
+	if err != nil {
+		if err.Error() == "pebble: not found" {
+			return &ConversionStatusResult{}, nil
+		}
```

Comment:
Should we care about LevelDB? Or in this case it doesn't return an error and `len(data)==0`?

Reply:
leveldb support is going to be dropped, I think we shouldn't worry about it